src/ui/components/Library/RecordingRow.tsx

```diff
@@ -29,6 +30,43 @@ export function getRelativeDate(date: string) {
   return content;
 }
 
+function RowWrapper({
+  children,
+  isEditing,
+  isOwner,
+  recording,
+  toggleChecked,
+}: {
+  children: React.ReactNode;
+  isEditing: boolean;
+  isOwner: boolean;
+  recording: Recording;
+  toggleChecked: () => void;
+}) {
+  const isTesting = new URL(window.location.href).searchParams.get("e2etest");
+  let url = `/recording/${recording.id}`;
+
+  if (isTesting) {
+    url += `?e2etest=true`;
+  }
+
+  const onClick = () => {
+    if (isEditing && isOwner) {
+      toggleChecked();
+    }
+  };
+
+  return (
+    <a
+      href={isEditing ? undefined : url}
```

Comment:
This seems.... a little weird?

Reply:
Bit jank. Let me see if there's a cleaner way...